frontstage/templates/partials/header.html

```diff
@@ -30,7 +30,7 @@
 
     {% if 'authorization' in request.cookies %}
     <div class="sign-out header__sign-out">
-      <a class="u-ml-s nowrap" href="{{ url_for('sign_in_bp.logout') }}">Sign out</a>
+      <a id="SIGN_OUT_BTN" class="u-ml-s nowrap" href="{{ url_for('sign_in_bp.logout') }}">Sign out</a>
```

Comment:
is there a reason this id is all caps?

Reply:
id Being all caps makes no difference, it depends on who adds it etc, there are some that are all caps and others that are not

Reply:
:+1: looks like there's no standards around capitalisation anyway so will be fine